api.js

```diff
@@ -0,0 +1,32 @@
+var movies = [];
```

Comment:
If this is truly just a test file, and not part of the working project, it should be removed.